script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_2.txt

```diff
@@ -23,7 +23,7 @@ The one that's controlling those zombies or something."
 <Page7>
 "Castle Lords and Ancestors are not necessarily the same.
 The more powerful a Dead Apostle is, the easier it is for them to become a Castle Lord, and a Castle Lord is something more like a local ruler.
```

Comment:
the easier it is for them to become a Castle Lord, who acts as something like a local ruler.

Reply:
a castle lord, which is something more like a local ruler.